vnext/Scripts/rnw-dependencies.ps1

```diff
@@ -201,11 +201,12 @@ $requirements = @(
         Install = { choco install -y yarn };
     },
     @{
-        Name = 'Appium';
+        Name = 'Python';
         Tags = @('rnwDev');
-        Valid = (Test-Path "${env:ProgramFiles}\Appium\Appium.exe");
-        Install = { choco install -y Appium-desktop };
-        Optional = $true;
+        Valid = try {
+            return (choco list -l python3 | where-Object { $_ -like 'python3 *' }) -ne $null
```

Comment:
Consider also testing against the result of `python3 --version` since python may be installed by other means.

Reply:
@NickGerleman the problem is that Windows ships a python3.exe, but it's just a stub to install python through a Store package :-/